Initial creating blueprint docs (#15816)
home-assistant/home-assistant.github.io: source/_docs/blueprint.markdown

---

> title: "Creating Blueprints"

Reviewer: ```suggestion
title: "Creating blueprints"
```

description: "Documentation on how to get started creating blueprints."
---

> <p class='note'>
> If you're looking on how to use blueprints, see the <a href="/docs/automation/using_blueprints/">automation documentation</a>.
> </p>

Reviewer: ```suggestion
<div class='note'>

If you're looking on how to use blueprints, see the [automation documentation](/docs/automation/using_blueprints/).

</div>
```

> An automation blueprint is an automation configuration with certain parts marked as configurable. This allows users to create multiple automations based on the same blueprint, with each having their own configuration.

Reviewer: ```suggestion
An automation blueprint is an automation configuration with certain parts marked as configurable. This allows users to create multiple automations based on the same blueprint, with each having its own configuration.
```

> Imagine a blueprint that controls a light based on motion that allows you to configure the motion sensor to trigger on, and the light to control. It is now possible to create two automations that each have their own configuration for this blueprint and that act completely independent, yet are based on the same automation configuration.

Reviewer: ```suggestion
Imagine a blueprint that controls a light based on motion, that allows you to configure the motion sensor to trigger on, and the light to control. It is now possible to create two automations that each have their own configuration for this blueprint and that act completely independent, yet are based on the same automation configuration.
```

> <p class='note'>
> This is an advanced feature and requires knowledge of writing <a href="/docs/automation/yaml/">automations in YAML</a>.
> </p>

Reviewer: ```suggestion
<div class='note'>

This is an advanced feature and requires knowledge of writing [automations in YAML](/docs/automation/yaml/).

</div>
```

> ### [Tutorial: Create a Blueprint &raquo;](/docs/blueprint/tutorial/)

Reviewer: ```suggestion
### [Tutorial: Create a blueprint &raquo;](/docs/blueprint/tutorial/)
```
